app/services/kdigo_preprocess.py

```
@@ -0,0 +1,91 @@
+    # 1. chunking
+    chunks = chunk_kdigo_text(kdigo_text, chunk_size, chunk_overlap)
+    if not chunks:
+        print("[KDIGO Preprocess] 청크가 생성되지 않았습니다.")
+        return 0
+
+    print(f"[KDIGO Preprocess] 생성된 청크 수: {len(chunks)}")
+
+    # 2. 임베딩
+    embeddings = embed_texts_with_gemini(chunks)
+    if len(embeddings) != len(chunks):
+        raise RuntimeError("청크 개수와 임베딩 개수가 일치하지 않습니다.")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)